src/TaskCardLayout/DoneCard.js

```diff
@@ -0,0 +1,12 @@
+export default function DoneCard({ doneTask }) {
+	return (
+		<article className="DoneCard">
+			<h2>
+				<span id="undo">{doneTask.done && "✅"}</span>
```

Comment:
@Tom-Sarfo I'm not sure what the condition of this `span` is. Isn't the purpose of this component to show items that are done?

Reply:
Yeah, @noramens   for done items. I actually brought the span there for styling purposes. I will double-check to see if it is still relevant to keep.